### ISSUE TITLE
bug: getClickPoint returning 0,0

### ISSUE THREAD

**Issue by Vitalflea** [unethicalite/unethicalite#125]
**Describe the bug**
getClickPoint function for TileObjects and NPCs is returning 0,0

**To Reproduce**
execute getClickPoint 

**Expected behavior**
returns valid clickpoint

**Environment (please complete the following information):**
Windows 10 19041
unethicalite 1.0.1-SNAPSHOT
development branch

**Comment by Vitalflea** [comment]
just realzied that it works as intended when using minimal mode.

**Comment by buracc** [comment]
seems to be working fine on my end, can u check if it still occurs?

**Comment by Vitalflea** [comment]
yes it still happens. i tried minimal on my main pc instead of 2nd pc and it was not working there either. so im guessing its environment related. when i try using breakpoints on the code for cached bounds for npcs/tileobjects in the mixins, it does not get executed like it does for widgets or other bounding/clickbox things

**Comment by Vitalflea** [comment]
so its getting reset, almost like how the mouse forwarding position was not being saved/loaded

![test46](https://user-images.githubusercontent.com/30124058/179450490-7cd1c722-8ef0-4296-99a0-3d9fc7f02c6c.gif)
